scripts/kubestellar

```diff
@@ -0,0 +1,221 @@
+#!/usr/bin/env bash
+# Copyright 2023 The KCP Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Usage: $0 (start or stop | -v)
+
+# Purpose: control whether the controllers are running, setting up the ESPW if necessary.. The following components are created:
+#           (a) 1 kcp workspace: edge service provider workspace (espw)
+#           (b) 3 kubestellar controllers: kubestellar-scheduler, mailbox-controller and placement-translator
+
+# Assumption: kcp server is running.
+
+# Requirements:
+#    Download KubeStellar binaries
+#    KubeStellar controller binaries are on $PATH.
+
+
+set -e
+
+install=""
+verbosity=0
+remove=0
+cleanup=0
+espw_name="espw"
+log_folder=$(pwd)/kubestellar-logs
```

Comment:
Perhaps `$(pwd)/kubestellar/logs` would be more consistent, making stuff alongside `bin`, `config`, and `examples`?

Reply:
this script works, and that is all we need for now.  Please push this change to another PR and we can debate it after we publish the first version of our quickstart.